cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -691,3 +691,80 @@ subjects:
   namespace: rook-ceph
 # OLM: END CMD REPORTER ROLEBINDING
 ---
+#
+# OLM: BEGIN CLUSTER ROLE
+---
+# Aspects for creation of monitoring resources
+kind: ClusterRole
```

Comment:
We could add this to an existing ClusterRole, I believe you want `rook-ceph-global-rules`. Or is there a need to create a new cluster role?

Reply:
There is no specific need. But, wouldn't it be better to keep it separate so that users who do not want monitoring can skip these rbac?